app/Insightly/Resources/InsightlyProjectResource.php

```diff
@@ -106,11 +121,11 @@ public function linkContact(int $projectId, int $contactId, ContactType $contact
         $this->insightlyClient->sendRequest($request);
     }
 
-    private function get(int $id): array
+    private function get(int $projectId): array
     {
         $request = new Request(
             'GET',
-            'Projects/' . $id
+            'Projects/' . $projectId
```

Comment:
I noticed we sometimes use the property `$this-path` and sometimes a hardcoded string. Best to make consistent use of the `$path` property. While checking this please also check the other resources.